dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/internal/resources/OAuthMessages.nlsprops

```diff
@@ -458,6 +458,20 @@ WRONG_TOKEN_GRANTTYPE=CWWKS1497E: The token with grant type [{0}] is not allowed
 WRONG_TOKEN_GRANTTYPE.explanation=The token is not valid because its grant type is not allowed.
 WRONG_TOKEN_GRANTTYPE.useraction=Use a token with a grant type that is allowed.
 
+OAUTH_CLIENT_REGISTRATION_VALUE_URI_CONTAINS_FRAGMENT=CWWKS1498E: The URI [{0}] for the client registration metadata field {1} is not valid because it contains a fragment.
```

Comment:
The messages are well written. That said, please use the "the {variable} noun" format unless the current "the noun {variable}" format is needed for clarity. Thus, change:
`The URI [{0}] for the client registration metadata field {1}`
to:
`The [{0}] URI for the {1} client registration metadata field `

This comment applies for all occurrences of the "the noun {variable}" format in the new messages.